yaqd_mcc/mcc-152-aout.toml

```diff
@@ -0,0 +1,39 @@
+protocol = "mcc-152-aout"
+doc = ""
+traits = ["has-position", "has-limits", "is-daemon"]
+
+[links]
+source = "https://github.com/yaq-project/yaqd-mcc"
+bugtracker = "https://gitlab.com/yaq/mccdaq/-/issues"
+
+[installation]
+PyPI = "https://pypi.org/project/yaqd-mcc"
+
+[config]
+
+[config.address]
+doc = "daqhat address"
+type = "int"
+default = 0
+
+[config.terminal]
+doc = "daqhat terminal"
+type = "int"
+
+[config.make]
```

Comment:
not necessarily wrong, but I will point out that only default needs to be specified, as type and doc of make/model are defined by traits (its fine, but not needed to specify the type and doc)

Reply:
actually, it might be wrong, as the CI failure is related to the `is-daemon` trait, where these two are defined

Reply:
I think the problem was that the type was narrowed from ["null", "string"] to "string", which is not currently allowed by yaq-traits check (See yaq-project/yaq-traits#1)

This is eventually desired behavior, but simply does not work currently for our tooling.